baybe/recommenders/pure/bayesian/base.py

```diff
@@ -47,75 +54,53 @@ def _validate_deprecated_argument(self, _, value) -> None:
                 "The parameter has been renamed to 'acquisition_function'."
             )
 
-    def _setup_botorch_acqf(
-        self,
-        searchspace: SearchSpace,
-        train_x: Optional[pd.DataFrame] = None,
-        train_y: Optional[pd.DataFrame] = None,
+    def _transform_data(
+        self, data: pd.DataFrame, searchspace: SearchSpace, objective: Objective
     ) -> None:
-        """Create the current acquisition function from provided training data.
+        """Transform data from experimental to computational representation."""
+        self._train_x = searchspace.transform(data)
```

Comment:
just for clarity: you intend this to be the place marko is supposed to add the augmentation?

Reply:
Ok, from my end the signature change is completed. Will only fix the remaining open threads but content-wise we're good. So once merged, the road is unblocked for @mhrmsn and we can discuss where and how exactly to incorporate the augmentation mechanism. Let me know if there is more to discuss here, otherwise we can move ahead 🚀